workers/unity/Assets/Fps/Scripts/GameLogic/RandomNameGenerator.cs

```diff
@@ -0,0 +1,70 @@
+﻿using UnityEngine;
+
+namespace Fps
+{
+    public static class RandomNameGenerator
+    {
+        public static string GetName()
+        {
+            var prefix = Prefixes[Random.Range(0, Prefixes.Length)];
+            var postfix = Postfixes[Random.Range(0, Postfixes.Length)];
+
+            var timeOut = 200;
+            while (postfix.Length + prefix.Length > 15 && timeOut > 0)
```

Comment:
Why not just make a character limit for postfix and prefix such that any combination will be 15 or less characters?

Reply:
Resolving as this script should be deprecated soon anyway.